src/versionbits.h

```diff
@@ -30,6 +30,15 @@ enum ThresholdState {
 // will either be NULL or a block with (height + 1) % Period() == 0.
 typedef std::map<const CBlockIndex*, ThresholdState> ThresholdConditionCache;
 
+struct BIP9DeploymentInfo {
```

Comment:
Possibly. @sdaftuar was thinking maybe we should move the bit assignments here, however...